### ISSUE TITLE
Possible enhancement of test strategy variable

### ISSUE THREAD

**Issue by FatBoyXPC** [janko/vim-test#408]
In #401 I added a new strategy for `shtuff`. In my own config before this was merged, I had `g:shtuff_receiver` as the variable. @jesseleite pointed out that I might want to make it scoped to this test plugin, so maybe something like `test#shtuff_receiver` or `test#shtuff#receiver`. What are your thoughts on this? I'm happy to submit another PR in the future to take care of this.

**Comment by janko** [comment]
Yes, the convention for strategy-specific variables would be the latter, `test#shtuff#receiver`. Feel free to send a PR 👍

**Comment by FatBoyXPC** [comment]
I'll do that soon!
